internal/core/metadata/init.go

```diff
@@ -26,29 +27,21 @@ import (
 	"github.com/edgexfoundry/edgex-go/internal/pkg/errorconcept"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/coredata"
-	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"
 	"github.com/edgexfoundry/go-mod-core-contracts/clients/types"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/notifications"
```

Comment:
Why was this import separated?

Reply:
- I have removed the blank line, and separated the imports for consistency. The convention followed (for ordering the imports) is as follows:
`language libraries`
`blank line`
`edgex-go libraries`
`blank line`
`go-mod-core-contracts libraries`
`blank line`
`third-party libraries`
`blank line`